frontend/src/components/contact/ContactManagerView/ContactFilterComponent/ContactFilterComponent.tsx

```diff
@@ -0,0 +1,157 @@
+import { InlineInput } from 'components/common/form/styles';
+import { IContactFilter } from 'components/contact/ContactManagerView/IContactFilter';
+import { Form, Formik } from 'formik';
+import React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { FaRegBuilding, FaRegUser } from 'react-icons/fa';
+import styled from 'styled-components';
+
+import ActiveFilterCheck from './ActiveFilterCheck';
+
+export const defaultFilter: IContactFilter = {
+  summary: '',
+  municipality: '',
+  searchBy: 'all',
+  activeContactsOnly: true,
+};
+
+export interface IContactFilterComponentProps {
+  filter?: IContactFilter;
+  setFilter: (filter: IContactFilter) => void;
+  showActiveSelector?: boolean;
+}
+
+/**
+ * Filter bar for contact list.
+ * @param {IContactFilterComponentProps} param0
+ */
+export const ContactFilterComponent: React.FunctionComponent<IContactFilterComponentProps> = ({
+  filter,
+  setFilter,
+  showActiveSelector,
+}: IContactFilterComponentProps) => {
+  const resetFilter = (values: IContactFilter) => {
+    setFilter({ ...defaultFilter, searchBy: values.searchBy });
+  };
+  return (
+    <Formik
+      enableReinitialize
+      initialValues={filter ?? defaultFilter}
+      onSubmit={(values, { setSubmitting }) => {
+        setFilter(values);
+        setSubmitting(false);
+      }}
+      validateOnChange={true}
+    >
+      {({ resetForm, isSubmitting, values, submitForm }) => (
+        <FilterBoxForm>
+          <Row>
+            <Col xs="auto">
+              <RadioGroup
+                label="Search by:"
+                field="searchBy"
+                radioGroupClassName="pb-3"
+                radioValues={[
+                  {
+                    radioLabel: (
+                      <>
+                        <FaRegBuilding size={20} />
+                        <span>Organizations</span>
+                      </>
+                    ),
+                    radioValue: 'organizations',
+                  },
+                  {
+                    radioLabel: (
+                      <>
+                        <FaRegUser size={20} />
+                        <span>Individuals</span>
+                      </>
+                    ),
+                    radioValue: 'persons',
+                  },
+                  {
+                    radioLabel: (
+                      <>
+                        <FaRegBuilding size={20} />+<FaRegUser size={20} />
+                        <span>All</span>
+                      </>
+                    ),
+                    radioValue: 'all',
+                  },
+                ]}
+              />
+            </Col>
+            <Col>
+              <Row>
+                <Col className="pl-0">
+                  <NameInput field="summary" placeholder="Name of person or organization" />
+                </Col>
+                <Col xs="auto" className="pl-0">
+                  <CityInput field="municipality" label="City" />
```

Comment:
one nice thing about Styled.CityInput or StyledCityInput is that it makes it clear that this is a display only component wrapper.